package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "matrix-rn-sdk",
```

Comment:
Don't want to bikeshed too hard here, but just curious... Is `rn` a clear enough name, or would spelling out `react-native` be better? What's common for React Native community packages?

Looking at e.g. [Ditto's deps](https://gitlab.com/ditto-chat/ditto-mobile/-/blob/dev/package.json#L20), they seem to all spell out `react-native`.

Reply:
Not entirely sure what the trends are: I copied Annie's 'rnmatrix' on the basis that matrix-rn-sdk put it lexically closer to matrix-js-sdk and less like matrix-react-sdk (ie. convern that it would sound like a react native version of matrix-react-sdk).